tools/go-changelog/note.go

```diff
@@ -48,7 +64,7 @@ func NotesFromEntry(entry Entry) []Note {
 				}
 			}
 
-			note = strings.TrimSpace(note)
+			// note = strings.TrimSpace(note)
```

Comment:
We need to skip this here so that we can detect changelogs that contain multiple lines

Reply:
```suggestion
```